### ISSUE TITLE
Build fails with swiftlint 0.29.1

### ISSUE THREAD

**Issue by benweitzman** [facebook/facebook-swift-sdk#300]
When running the latest version of swiftlint, it is an error to have a different version named in `.swiftlint.yml`: https://github.com/realm/SwiftLint/pull/2491, https://github.com/realm/SwiftLint/issues/2074

## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-swift/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `10.1`
- Swift Version: `4.2.1`
- Installation Platform & Verison: `Carthage` version `0.31.2`
- swiftlint version: `0.29.1`

## Actual Results

```
FacebookCore/IntermediateBuildFilesPath/FacebookSwift.build/Release-iphoneos/FacebookCore.build/Script-5257097220F6C9800053B9E3.sh
Loading configuration from '.swiftlint.yml'
Currently running SwiftLint 0.29.1 but configuration specified version 0.26.0.
Command PhaseScriptExecution failed with a nonzero exit code
```

**Comment by codytwinton** [comment]
@benweitzman - Thanks for bringing this to our attention.

In the meantime, we're open to accepting any PRs to resolve this issue.

**Comment by codytwinton** [comment]
@beeloo42 - This has been resolved in master with #304

**Comment by nicolas-miari** [comment]
@codytwinton 
I don't understand; the line:

    swiftlint_version: 0.26.0

...is clearly removed from the top of [.swiftlint.yml](https://github.com/facebook/facebook-swift-sdk/blob/master/.swiftlint.yml); However, when I clone 0.5.0 withv Carthage, it's still there...?
